samples/bot-daily-task-reminder/nodejs/.env

```diff
@@ -1,3 +1,5 @@
 MicrosoftAppId=
 MicrosoftAppPassword=
-BaseUrl=
\ No newline at end of file
+BaseUrl=
+MicrosoftAppType=
```

Comment:
remove this 2 settings, if not required. Upate teamsapp.local.yml file also

Reply:
fixed